### PR TITLE
Add the max_uint subcommand to cast and cli

### PR DESCRIPTION
**Pull request by hammeiam**
Screen Shot 2021-12-13 at 12 16 38 AM

### CODE REVIEW

File: cli/src/cast_opts.rs [hunk 1]
```diff
@@ -12,6 +12,9 @@ use structopt::StructOpt;
 #[derive(Debug, StructOpt)]
 #[structopt(about = "Perform Ethereum RPC calls from the comfort of your command line.")]
 pub enum Subcommands {
+    #[structopt(name = "--max-uint")]
+    #[structopt(about = "maximum u256 value")]
+    MaxUint { },
```
Comment (mattsse) [inline cli/src/cast_opts.rs:17]:
```suggestion
    MaxUint,
```
this should still work I suppose?

File: cli/src/cast.rs [hunk 2]
```diff
@@ -18,6 +18,9 @@ async fn main() -> eyre::Result<()> {
 
     let opts = Opts::from_args();
     match opts.sub {
+        Subcommands::MaxUint { } => {
```
Comment (mattsse) [inline cli/src/cast.rs:21]:
```suggestion
        Subcommands::MaxUint => {
```

**Review by mattsse — APPROVED** [review]
lgtm

File: cli/src/forge.rs [hunk 3]
```diff
@@ -1,4 +1,7 @@
-use ethers::solc::{remappings::Remapping, Project, ProjectPathsConfig};
+use ethers::{
+    prelude::{ArtifactOutput, ContractFactory, MinimalCombinedArtifacts, Provider},
```
Comment (hammeiam) [inline cli/src/forge.rs:2]: Since these imports are unused, they seem to cause clippy to return a warning

**Review by gakonst — APPROVED** [review]
LGTM

**Pull request merged**